app/controllers/api/v1/people_actions.rb

```diff
@@ -62,7 +62,7 @@ def updater
     end
 
     def filter_params
-      params.require(:filter).permit(PERMITTED_FILTER_PARAMS).to_h
+      params.fetch(:filter, {}).permit(PERMITTED_FILTER_PARAMS).to_h
```

Comment:
do you also need to update v2 people_actions ?

Reply:
(and also `spec/requests/api/people_controller_index_v2_spec.rb`)?

Reply:
> do you also need to update v2 people_actions ?
That one is correct, but now there is a bit of duplication that can be removed.